gradle/verification-metadata.xml

```diff
@@ -1804,6 +1804,11 @@
             <sha256 value="dd8e7c92185a678d1b7b933f31209b6203c8ffa91e9880475a1be0346b9617e3" origin="Generated by Gradle"/>
```

Comment:
should we remove the previous 2.10.10 component part?

Reply:
Good idea